build.sh

```diff
@@ -48,6 +48,7 @@ _init() {
 
     # Set the package versions string
     PG_PACKAGE_VERSION=$PG_MAJOR_VERSION.`echo $PG_MINOR_VERSION | sed -e 's/\./-/'`
+    PG_PACKAGE_MINOR_VERSION=`echo $PG_MINOR_VERSION | awk -F'.' '{print $1}'`
```

Comment:
how does it behave during beta/rc?

Reply:
it should show minor version as 0 during beta/rc as we update minor version as 0.beta1 or 0.rc1 etc. This is in the case when we build through traditional method

Reply:
@sandeep-edb